### PR TITLE
fix broken tests

### PR DESCRIPTION
**Pull request by prolic**
(because of phpunit-util changes)

### CODE REVIEW

**Comment by prolic** [comment]
I hope I catched all places.

**Comment by enumag** [comment]
@prolic Apparently not. Still failing.

**Comment by prolic** [comment]
Okay, that should do it. Feel free to merge @enumag, the other pull requests just need to be rebased after that, so tests pass there as well.
If it still ain't working, I'll have another look tomorrow

File: tests/EventStoreConnectionTestCase.php [hunk 1]
```diff
@@ -28,8 +29,10 @@ protected function setUpAsync(): Generator
         yield $this->connection->connectAsync();
     }
 
-    protected function tearDownAsync()
+    protected function tearDownAsync(): Generator
     {
         $this->connection->close();
+
+        yield new Success();
```
Comment (enumag) [inline tests/EventStoreConnectionTestCase.php:36]: why is this necessary? :thinking:

Comment (enumag) [reply tests/EventStoreConnectionTestCase.php:36]: Ah I think I understand now.

Comment (prolic) [reply tests/EventStoreConnectionTestCase.php:36]: Because then it's not a generator. tearDown (non-async) is called later, so the test would hang otherwise forever (unless you get to unreference watchers).

Comment (prolic) [reply tests/EventStoreConnectionTestCase.php:36]: If unreferencing the connection works, then the method can be deleted here anyway, as we don't need to close the connection anymore.

**Comment by prolic** [comment]
Tests pass, yeah! Good night!

**Comment by enumag** [comment]
Works like a charm. Thanks and good night!
